src/Symfony/Bundle/FrameworkBundle/Resources/config/services.php

```diff
@@ -47,7 +48,6 @@
 use Symfony\Component\HttpKernel\HttpKernelInterface;
 use Symfony\Component\HttpKernel\KernelEvents;
 use Symfony\Component\HttpKernel\KernelInterface;
-use Symfony\Component\HttpKernel\UriSigner;
```

Comment:
This removes the current service alias which is a BC break, it should be deprecated instead, see #46602

Reply:
do we still require the alias when `Symfony\Component\HttpKernel\UriSigner` is an class alias to `Symfony\Component\HttpFoundation\UriSigner`?

Reply:
I readded the alias as ReflectionParameter would return from as `->getType()->__toString()` still the old class and I'm not sure if that kind of reflection is used for autowiring.

Reply:
The alias should be deprecated though.

Reply:
I dont see a way to deprecate here as it returns the parent again?